packages/components/src/radio-control/styles/radio-control-styles.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { breakpoint, config, color } from '../../utils';
+import { inputControl } from '../../utils/input';
+import {
+	StyledHelp as BaseControlHelp,
+	StyledField as BaseControlField,
+} from '../../base-control/styles/base-control-styles';
+import BaseControl from '../../base-control';
+
+export const RadioControlWrapper = styled( BaseControl )`
+	display: flex;
+	flex-direction: column;
+
+	${ BaseControlHelp } {
+		margin-top: 0;
+	}
+
+	${ BaseControlField } {
+		margin-bottom: 0;
+	}
+`;
+
+export const RadioControlOption = styled.div`
+	&:not( :last-child ) {
+		margin-bottom: 4px;
+	}
+`;
+
+export const RadioControlInput = styled.input`
+	${ inputControl }
+
+	border: ${ config( 'borderWidth' ) } solid ${ color( 'gray.900' ) };
+    margin-top: 0;
+	margin-right: 6px;
+	transition: none;
+	border-radius: ${ config( 'radiusRound' ) };
+
+	&:checked::before {
+		width: 7px;
+		height: 7px;
+		margin: 8px 0 0 8px;
+		background-color: ${ color( 'white' ) };
+
+		// This border serves as a background color in Windows High Contrast mode.
+		border: 3px solid ${ color( 'white' ) };
+
+		${ breakpoint( 'medium' ) } {
+			width: 6px;
+			height: 6px;
+			margin: 4px 0 0 4px;
+		}
+	}
+
+	&:focus {
+		box-shadow: 0 0 0 calc( ${ config( 'borderWidth' ) } * 2 ) ${ color(
+	'white'
+) },
+			0 0 0 calc( ${ config( 'borderWidth' ) } * 2 + ${ config(
+	'borderWidthFocus'
+) } )
```

Comment:
`prettier` sure did a number on these lines. Not sure if it's worth trying to do anything about it.

Reply:
REKT. I'm seeing that on my side as well. Prettier is super unhappy with this. I don't blame it... there are a lot of things going on 🤣 

Maybe we can create a mixin for it. It'll make it easier to read (kinda), but it should make the linters happier.

Something like this:

```js
const radioFocusBoxShadow = () => {
	const boxShadow = [
		`0 0 0 calc( ${ config( 'borderWidth' ) } * 2 ) ${ color( 'white' ) }`,
		`0 0 0 calc( ${ config( 'borderWidth' ) } * 2 + ${ config(
			'borderWidthFocus'
		) } ) var( --wp-admin-theme-color )`,
	].join( ',' );

	return css( { boxShadow } );
};
```

Reply:
It looks like the linter is still somewhat confused by this. I guess I don't blame it.
Totally not your fault! This makes me think that we need a better strategy (maybe) to handle complex string interpolation for CSS shorthands especially if it involves functions like `calc()`.